client/views/msg_preview_toggle.tpl

```diff
@@ -0,0 +1,10 @@
+{{#preview}}
+<button class="toggle-button {{#if shown}} opened{{/if}}"
+  data-url="{{link}}"
+  {{#equal type "image"}}
+    aria-label="Toggle image preview"
```

Comment:
See https://github.com/thelounge/lounge/pull/1303/files#diff-22e42391cd970f0b0b44672abb685a0aL3, and no tooltips in UI right now.